### DISCUSSION TITLE
EC2 Instance for Wiki Entity Linker

### DISCUSSION THREAD

**Discussion by sednaasil** [explosion/spaCy#7180] [Help: Coding & Implementations] [answered]
Hi:

I was wondering if anyone had a recommendation for EC2 instance size (e.g. small,medium,large) and available memory for the wiki entity linker development?  Any recommendations would be really helpful.

v/r,
L

**Comment by svlandeg** [comment, marked as answer]
I haven't run this code recently, but [this comment](https://github.com/explosion/spaCy/issues/4511#issuecomment-566074900) (and perhaps the surrounding discussion) would be useful to you:

> On a machine with 16GB, it now takes me about 10h to run one epoch with 50.000 WP articles (`-t 50000`). Note that `-t` and `-d` have been modified and now refer to # of ARTICLES and not # of entities, so 50.000 is already quite a big training set !

**Reply by sednaasil** [reply]
Great!  Thanks so much :)
